From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Fri, 12 Jan 2018 18:08:14 +0100
Subject: shellutil.py: ignore tilde '~' as special character

Currently we don't need the tilde sign defined as a special character
inside the Python wrappers around the autotools files, it's not used any
there and breaks the build of Debian beta versions which uses '~' to
mark the Debian version as 'smaller than'.
---
 python/mozbuild/mozbuild/shellutil.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/python/mozbuild/mozbuild/shellutil.py b/python/mozbuild/mozbuild/shellutil.py
index 185a970..580d778 100644
--- a/python/mozbuild/mozbuild/shellutil.py
+++ b/python/mozbuild/mozbuild/shellutil.py
@@ -26,7 +26,7 @@ UNQUOTED_TOKENS_RE = _tokens2re(
   whitespace=r'[\t\r\n ]+',
   quote=r'[\'"]',
   comment='#',
-  special=r'[<>&|`~(){}$;\*\?]',
+  special=r'[<>&|`(){}$;\*\?]',
   backslashed=r'\\[^\\]',
 )
 
